core/platform/search/gae_search_services.py

```diff
@@ -172,8 +173,11 @@ def _validate_list(key, value):
     passed in to make better error messages.
 
     Args:
-        key: str. key value of the list.
-        value: list. list to be included in document fields.
+        key: str. Key value string that fills in the descriptor of this
+            particular list.
+        value: list. List to be validated where elements must be either a
```

Comment:
list(*). The list to be validated. Each element of a valid list must be either a ...